Launcher/Classes/Language.swift

```diff
@@ -11,6 +11,10 @@ import Foundation
 enum Language: String {
     case english = "English"
     case german = "German"
+    case russian = "Russian"
```

Comment:
Using the actual strings will disallow you to localize the calabash launcher (if wanted) in the future. At least without changes. Maybe it’s not so important right now if it works. I would anyways fill the selection from code instead of the story board (if I see correctly, I’m doing this on the phone)

Reply:
If you filled the items from code you will be able to set the title independently from the value of the item.